packages/morbig/morbig.0.11.0/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+
+synopsis: "A trustworthy parser for POSIX shell"
+description: """
+Morbig is a parser for shell scripts written in the POSIX shell script
+language. It parses the scripts statically, that is without executing
+them, and constructs a concrete syntax tree for each of them. The
+concrete syntax trees are built using constructors according to the
+shell grammar of the POSIX standard.
+"""
+
+maintainer: "Nicolas Jeannerod <niols@niols.fr>"
+authors: [
+  "Yann Régis-Gianas <yann.regis-gianas@irif.fr>"
+  "Ralf Treinen <ralf.treinen@irif.fr>"
+  "Nicolas Jeannerod <niols@niols.fr>"
+]
+license: "GPL-3.0-only"
+
+homepage: "https://github.com/colis-anr/morbig"
+bug-reports: "https://github.com/colis-anr/morbig/issues"
+dev-repo: "git+https://github.com/colis-anr/morbig.git"
+
+depends: [
+  "dune"                 {>= "1.4.0"}
+  "menhir"               {>= "20180538"}
+  "ocaml"                {>= "4.04"}
+  "odoc"                 {with-doc}
+  "ppx_deriving_yojson"
+  "visitors"             {>= "20180513"}
+  "yojson"               {>= "1.6.0" & < "2.0.0"}
+]
+
+build: ["dune" "build" "-p" name "-j" jobs]
```

Comment:
I notice your `Makefile` has a special build rule for when `ocamlopt` is available. The build rule in this opam file will not use that special `dune.native` trick.

Is that important?

Reply:
Oh, you're absolutely right. I will need to take a look at the newer version of Dune and see if I can't encode this trick in the configuration now. Without copying this `dune.native` to `dune`, we don't build the C library. I am not sure it is so problematic, but let me push build instructions that go through the `Makefile` anyways.